spec/board_spec.rb

```diff
@@ -0,0 +1,66 @@
+require './lib/board'
+
+describe "Board Class" do
+  let(:board) {Board.new}
+
+  it "should have board class" do  
+    expect(board.class).to eql(Board)
+  end
+  
+  describe "#update_board" do   
+    it "should change the board cell number to character" do
+      expect(board.update_board(1, "X")).to eql(board.board[0])
```

Comment:
You should do a test for incorrect number and number in space already taken.

That should give you 2 more it statements

Reply:
Hey Maya, our update board method doesn't check if the given number is correct or if the cell is already taken, there are other methods which do that and we are testing them.
`def check_sign(num) `
     `@board[num.to_i - 1] == num.to_i`
 `end`
this method, for instance, is checking if the cell is already taken.
so we are confused about why we should test update_board method with incorrect number and number in space already taken.